.github/workflows/build.yaml

```diff
@@ -0,0 +1,43 @@
+name:  Build protobuf stubs
+
+on:
+  push:
+    paths:
+      - src/**
+      - CMakeLists.txt
+      - .github/**
+  workflow_dispatch:
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: "16"
+    #- name: Install additional tooling
+    #  run: |
+    #    sudo apt install -y protobuf-compiler-grpc libgrpc6 libgrpc++1
+    - uses: eWaterCycle/setup-grpc@v5
+    - name: Install Protoc
+      uses: arduino/setup-protoc@v1
+      #with:
+      #  version: '3.6'
+    - name: install pip dependencies
+      run: pip3 install grpcio-tools
+    - name: CMake & Build 
+      run: |
+        cmake .
+        cmake --build .
+    - name: Commit files
+      run: |
+        git config --local user.name "dmarx"
```

Comment:
This should not override the default action user.

Reply:
this will run into the same issue described in the sdk CI: pretty sure if I don't configure a user, it'll just complain. How do we make sure a default action user is configured? or should we just choose a consistent username for gh actions?

Reply:
Same as there: 
> You can use github.actor from the github context https://docs.github.com/en/actions/learn-github-actions/contexts#github-context.